readme.md

```diff
@@ -102,7 +102,7 @@ Once installation is working, take a look at the existing code to make sure you
 
   _NOTE: We will not go into this too deeply for the sake of brevity, however this is a really interesting topic! I would encourage you to look into this more on your own, if you're interested._
 
-* **Your Answer:**
+* **Your Answer:** Salt rounds will set a number or rounds that will be used to hash the password.
```

Comment:
Yes, more rounds creates a more secure hash but also exponentially increases the server resources required.